### PR TITLE
feat: image and input components added

### PR DESCRIPTION
**Pull request by Rishabh-078**
Images: This component contains responsive image and round image
Input: This component included standard Input box as well as input with error, success and warning validations
screenshot-127 0 0 1_5501-2022 02 13-12_52_30
.

### CODE REVIEW

File: lib/components/Images/image.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
+}
+
+.responsive-img {
+    width: 100%;
+    height: 30rem;
+}
+
+.round-image-container {
```
Comment (manish-GH) [inline lib/components/Images/image.css:24]: TIP: You could use 

display: grid; 
place-content: center

shorter method to center stuff

File: lib/components/Images/image.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
+}
+
+.responsive-img {
+    width: 100%;
+    height: 30rem;
```
Comment (yaswanthmyneni) [inline lib/components/Images/image.css]: add object-fit property and give height 100% to images, according to the image container the images should be responsive.

File: lib/components/Images/image.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.round-img {
+    border-radius: 50%;
+    width: 25rem;
+    height: 25rem;
```
Comment (yaswanthmyneni) [inline lib/components/Images/image.css]: same here too, give width and height as 100% to images. also, add the aspect ratio property in the round image.

File: lib/components/Images/image.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.image-component iframe{
+    height: 8rem;
+}
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
```
Comment (yaswanthmyneni) [inline lib/components/Images/image.css:13]: give a specific width and height here, not in images itself.

**Review by manish-GH — APPROVED** [review]
It looks weird having a purple box-shadow for one image and a grey box-shadow for the other. Try and have just one color.

Individual components are nice. Can be merged

File: lib/components/Input/input.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+.input-wrapper iframe{
+    height: 15rem;
+}
+
+.input{
+    font-size: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 0.75rem;
```
Comment (yaswanthmyneni) [inline lib/components/Input/input.css]: use px for units less than 1rem.

**Comment by yaswanthmyneni** [comment]
You can merge after changes.

File: lib/components/Images/image.css (hunk 2 above)
Comment (manish-GH) [reply lib/components/Images/image.css]: He's right, I didn't notice it. You'll need to change this

Comment (Rishabh-078) [reply lib/components/Images/image.css]: Thanks for letting me know, had made the required changes

File: lib/components/Images/image.css (hunk 3 above)
Comment (Rishabh-078) [reply lib/components/Images/image.css]: Made the required changes

File: lib/components/Images/image.css (hunk 4 above)
Comment (Rishabh-078) [reply lib/components/Images/image.css:13]: I understood your point and implemented it.

File: lib/components/Images/image.css (hunk 1 above)
Comment (Rishabh-078) [reply lib/components/Images/image.css:24]: I didn't knew about this, thanks for the advice

File: lib/components/Input/input.css (hunk 5 above)
Comment (Rishabh-078) [reply lib/components/Input/input.css]: Corrected it.

**Pull request merged**